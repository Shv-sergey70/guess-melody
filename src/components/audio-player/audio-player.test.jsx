import React from 'react';
import {configure, mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import AudioPlayer from "./audio-player";

configure({adapter: new Adapter()});

test(`AudioPlayer correctly renders`, () => {
  const tree = mount(
      <AudioPlayer
        src="/my/custom/way/to/music.mp3"
        isPlaying={false}
        onPlayButtonClick={jest.fn()} />
  );

  expect(tree).toMatchSnapshot();
});
