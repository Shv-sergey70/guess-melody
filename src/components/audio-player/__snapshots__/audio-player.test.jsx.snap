// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer correctly renders 1`] = `
<AudioPlayer
  isPlaying={false}
  onPlayButtonClick={[MockFunction]}
  src="/my/custom/way/to/music.mp3"
>
  <button
    className="track__button track__button--play"
    disabled={true}
    onClick={[Function]}
    type="button"
  />
  <div
    className="track__status"
  >
    <audio
      src="/my/custom/way/to/music.mp3"
    />
  </div>
</AudioPlayer>
`;
